src/assets/Standards/ERC20Standard.test.ts

```diff
@@ -71,4 +72,48 @@ describe('ERC20Standard', () => {
     );
     expect(maticDecimals.toString()).toStrictEqual('18');
   });
+
+  it('should support non-standard ERC20 symbols and decimals', async () => {
+    nock('https://mainnet.infura.io:443', { encodedQueryParams: true })
+      .post('/v3/341eacb578dd44a1a049cbc5f6fd4035', {
+        jsonrpc: '2.0',
+        id: 3,
+        method: 'eth_call',
+        params: [
+          {
+            to: '0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2',
```

Comment:
[nit] - I know we don't do this in the other test but we should.

```suggestion
            to: MKR_ADDRESS,
```